packages/verified-fetch/test/abort-handling.spec.ts

```diff
@@ -62,6 +72,29 @@ describe('abort-handling', function () {
         return getAbortablePromise(options.signal)
       })
     })
+    unixFsCatStub.callsFake((cid, options) => {
+      unixFsCatCalled.resolve()
+      return {
+        async * [Symbol.asyncIterator] () {
+          await getAbortablePromise(options.signal)
+          yield new Uint8Array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+        }
+      }
+    })
+
+    unixFsStatStub.callsFake(async (cid, options): Promise<UnixFSStats> => {
+      unixFsStatCalled.resolve()
+      await getAbortablePromise(options.signal)
+      return {
+        cid,
+        type: 'file',
+        fileSize: BigInt(0),
+        dagSize: BigInt(0),
+        blocks: 1,
+        localFileSize: BigInt(0),
+        localDagSize: BigInt(0)
+      }
+    })
```

Comment:
defaults that timeout.... override with new `.callsFake` if you want it to return something valid.